tests/jballerina-unit-test/src/test/resources/test-src/expressions/stamp/map-stamp-expr-test.bal

```diff
@@ -274,27 +262,3 @@ function stampMapToRecordNegative() returns EmployeeClosedRecord|error {
 
     return employee;
 }
-
-function testStampRecordToRecordWithCyclicValueReferences() returns Engineer|error {
```

Comment:
Why did we removed these tests?

Reply:
I removed these 3 tests because they are the same as `testConvertRecordToRecordWithCyclicValueReferences`, `testConvertRecordToMapWithCyclicValueReferences`, `testConvertRecordToJsonWithCyclicValueReferences` tests in `NativeConversionNegativeTest` class.